sourcegraph/sourcegraph
# JetBrains: ensure active account is set

Previously it was possible to have a state when we had some accounts defined after migration but none of them were selected as active, now we added one more step (not dependent on the migration) after project startup to select first (preferably dotcom) account as active

closes #56566 

## Test plan
- Run migration from stable version to new version, all of previously configured access tokens for instances should be migrated to accounts and previously selected instance type should be transformed to active account
- Open completely new project after installing new version (you have some accounts defined) and we should automatically select first (preferably dotcom) account as active account
